src/components/Divider.js

```diff
@@ -32,11 +35,18 @@ export const Divider = ({ thoughtsRanked }) => {
 
   useEffect(setStyle)
 
-  return (<div ref={dividerSetWidth} style={{ width: '85px', maxWidth: '100%' }} className='divider-container'>
+  return <div ref={dividerSetWidth} style={{
+    margin: '-2px -4px -5px',
+    maxWidth: '100%',
+    padding: '10px 4px 16px',
+    position: 'relative',
+    width: 85,
+    zIndex: 1,
+  }} className='divider-container' onClick={setCursorToDivider}>
```

Comment:
@Ogutu-Brian Since we already have a container, it's better to add styles to that then introduce another container when there is no semantic distinction to draw.